website/docs/r/cdn_frontdoor_firewall_policy.html.markdown

```diff
@@ -298,6 +312,40 @@ An `exclusion` block supports the following:
 
 -> **Note:** `selector` must be set to `*` if `operator` is set to `EqualsAny`.
 
+---
+
+A `scrubbing_rule` block supports the following:
```

Comment:
This is just `rule` in the schema?
```suggestion
A `rule` block supports the following:
```

Reply:
Yes, but `managed_rule` also uses `rule` so I gave it a different name in the documentation trying to not confuse the end user when they are looking up what the `scrubbing_rule` block contains.

Reply:
Then I think we'll need to name it as such in the schema (as much as that will be an inconsistency / break with naming convention). Having these names different between the schema and the docs will be confusing/frustrating for users as they'll not know the correct name to use?